Cargo.toml

```diff
@@ -27,3 +27,6 @@ once_cell = "1.13"
 humantime = "2.1.0"
 rand = "0.8.5"
 chrono = "0.4.26"
+syn = { version = "2.0.60", features = ["full"] }
+ahash = { version = "0.8.9" }
```

Comment:
where and why is this being used?

Reply:
syn? syn is being used for parsing.
ahash is a crate that is used downstream and it had a bug that was causing my nightly to break.

Reply:
`cargo update` fixes ahash brokenness

Reply:
it wasnt doing it for me, for whatever reason.